File: code_wars/kyu_6/003-Your_order_please.js
// ###### JS ######
// ###### Solution 1
function order(words) {
  let words_split = words.split(" "),
    result = [...Array(words_split.length)];

  for (let i = 0; i < words_split.length; i++) {
    let num = words_split[i].match(/\d/)[0];
    result[Number(num)] = words_split[i];
  }

  return result;
}

// ######

File: code_wars/kyu_6/003-Your_order_please.py
###### Python ######
###### Solution 1


def order(sentence: str) -> str:
    """"""
    if not sentence:
        return ""

    words_split = sentence.split(" ")
    result = [None] * len(words_split)

    for word in words_split:
        for ch in word:
            if ch.isdigit():
                result[int(ch) - 1] = word
                break

    return " ".join(result)


######
